flixel/group/FlxSpriteGroup.hx

```diff
@@ -280,6 +281,8 @@ class FlxTypedSpriteGroup<T:FlxSprite> extends FlxSprite
 		sprite.alpha *= alpha;
 		sprite.scrollFactor.copyFrom(scrollFactor);
 		sprite.cameras = _cameras; // _cameras instead of cameras because get_cameras() will not return null
+		
+		if (clipRect != null) clipRectTransform(sprite, clipRect);
```

Comment:
Actually no, that would override the child's existing clip. The transform sets the child's clip based on the parent's, so you'd be setting `null` instead of leaving the existing one alone.